### PR TITLE
Redux setup

### PR DESCRIPTION
**Pull request by jackcarlisle**
* resolves merge conflicts 
* adds redux to the existing components

### CODE REVIEW

File: src/containers/home.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  showAddMessage: PropTypes.func,
+  hideAddMessage: PropTypes.func,
+  yesFilter: PropTypes.func,
+  noFilter: PropTypes.func,
+  filterVisible: PropTypes.boolean
+};
+
+function mapStateToProps (state) {
+  return ({
+    addMessageVisible: state.home.addMessageVisible,
```
Comment (nikhilaravi) [inline src/containers/home.js]: You could make this a bit more terse by using destructuring:
e.g. 
```js
 const { filter: { showAddMessage, hideAddMessage etc etc },  home: { addMessageVisible } } = state
```

File: src/reducers/tags.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
+'use-strict';
```
Comment (nikhilaravi) [inline src/reducers/tags.js]: Is this still a separate reducer? Can we get rid of this file?

File: src/components/home/index.js [hunk 3]
```diff
@@ -42,15 +28,15 @@ class Gallery extends Component {
...
+    const { addMessageVisible, showAddMessage, hideAddMessage, yesFilter, noFilter, filterVisible } = this.props;
     return (
       <div className='homeContainer'>
         <Modal showModal={this.state.modalVisible} close={this.closeModal}/>
         <SearchBar />
         <SearchSummary {...searchSummary} />
         <Tags />
-        <Grid showAddMessage={this.showAddMessage} tileData={tileData}/>
-        {addMessageVisible && <AddMessage hideAddMessage={this.hideAddMessage} suggestedLocations='Croatia and Greece'/>}
+        <Grid yesFilter={yesFilter} noFilter={noFilter} filterVisible={filterVisible} showAddMessage={showAddMessage} tileData={tileData}/>
```
Comment (nikhilaravi) [inline src/components/home/index.js]: You could give grid a container and make it a smart component to avoid passing the actions down.

**Comment by nikhilaravi** [comment]
@nelsonic @jackcarlisle please take a look! I've also set up the deployment pipeline to go to codeship and then to the www.tcdl.io bucket with the bucket folder being the version from the package.json

**Comment by nelsonic** [comment]
Cool. looks good. Look forward to seeing this progress further. :+1: 
Merging. :rocket:

**Pull request merged**